backend-api/app/schemas/users.py

```diff
@@ -4,7 +4,7 @@
 
 
 class UserRead(schemas.BaseUser[uuid.UUID]):
-    pass
+    eth_address: str | None
```

Comment:
If we are using the /users/me/wallet endpoint to get the eth_address and balance, we probably don't need to show it here.